samples/matter/lock/sample.yaml

```diff
@@ -28,28 +24,44 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.lock.smp_dfu_ffs:
+  sample.matter.lock.smp_dfu:
```

Comment:
Do we need this if we have `smp_dfu_ffs` already?

Reply:
In memory requirements table there is DL with SMP DFU and without FFS, it was a mistake that I removed it in my previous PR, didn't think about memory requirements report.

Reply:
I think there is no need to change the memory req table.